client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.html

```diff
@@ -0,0 +1,76 @@
+<div class="text-center"
+     data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div class="row">
+    <div class="col-md-8"
+         data-ng-if="!$ctrl.loading">
+
+        <h2 class="oui-heading_1"
+            data-translate="dedicatedCloud_tab_drp"></h2>
+        <oui-collapsible class="ml-0 mb-5"
+                         aria-label="{{:: 'dedicatedCloud_datacenter_drp_subtitle' | translate }}"
```

Comment:
`aria-label` is superseeded by `ui-kit`
```suggestion
                         data-aria-label="{{:: 'dedicatedCloud_datacenter_drp_subtitle' | translate }}"
```